docs/source/1.0/guides/building-models/build-config.rst

```diff
@@ -862,6 +862,49 @@ key is not in the provided ``keys`` list.
             }
         }
 
+.. _flattenNamespaces:
+
+flattenNamespaces
```

Comment:
I don't see how this would work if there are multiple services in a semantic model. They'd all be flattened into a single namespace? Can we instead require that a `service` property` is provided?

Can we also add the ability to "export" shapes? It could be done by selector or by tags.

Reply:
Updated the transformer.  A `service` property must be configured and an optional `includeTagged` property can now be specified to include additional shapes in the flattening.